packages/core/webpack.config.js

```diff
@@ -13,7 +13,6 @@ module.exports = {
     rules: [
       {
         test: /\.js$/,
-        exclude: /node_modules/,
```

Comment:
Does this mean that we're now using babel to rebuild all of our npm dependencies?